QCopycanvas.py

```diff
@@ -185,25 +186,11 @@ def unload(self):
                 action)
             self.iface.removeToolBarIcon(action)
 
-    def showMessage(self, title, msg, button):
-        msgBox = QMessageBox()
-        msgBox.setIcon(QMessageBox.Information)
-        msgBox.setWindowTitle(title)
-        msgBox.setText(msg)
-        msgBox.setStandardButtons(QMessageBox.Ok)
-        font = QFont()
-        font.setPointSize(9)
-        msgBox.setFont(font)
-        msgBox.setWindowFlags(Qt.CustomizeWindowHint | Qt.WindowStaysOnTopHint | Qt.WindowCloseButtonHint)
-        buttonY = msgBox.button(QMessageBox.Ok)
-        buttonY.setText(button)
-        buttonY.setFont(font)
-        msgBox.exec_()
-
     def run(self):
         """Run method that performs all the real work"""
 
         # Create the dialog with elements (after translation) and keep reference
         # Only create GUI ONCE in callback, so that it will only load when the plugin is started
         QApplication.clipboard().setImage(QImage(QWidget.grab(self.iface.mapCanvas())))
-        self.showMessage(title='QCopycanvas', msg='Copied map canvas successfully.', button='OK')
\ No newline at end of file
+        #self.showMessage(title='QCopycanvas', msg='Copied map canvas successfully.', button='OK')
+        self.iface.messageBar().pushMessage('QCopycanvas','Copied map canvas to clipboard', level=Qgis.Success,)
```

Comment:
What is that last comma in the pushMessage() function doing? Should it have a duration too?